packages/esm-active-visits-app/src/routes.json

```diff
@@ -4,6 +4,24 @@
     "webservices.rest": "^2.2.0"
   },
   "extensions": [
+    {
+      "name": "homepage-tiles",
+      "slot": "home-metrics-tiles-slot",
+      "component": "homePageTilesSlot",
+      "order": 0
```

Comment:
Did u implement order logic for tiles? if not then this isn't necessary. That logic is typically used in the patient chart nav